<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScreen</class>
 <widget class="QMainWindow" name="LoginScreen">
  <property name="windowTitle"><string>Login</string></property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="line_username">
    <property name="placeholderText"><string>Usu√°rio</string></property>
    <property name="geometry"><rect><x>100</x><y>40</y><width>200</width><height>30</height></rect></property>
   </widget>
   <widget class="QLineEdit" name="line_password">
    <property name="echoMode">2</property>
    <property name="placeholderText"><string>Senha</string></property>
    <property name="geometry"><rect><x>100</x><y>80</y><width>200</width><height>30</height></rect></property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="text"><string>Entrar</string></property>
    <property name="geometry"><rect><x>150</x><y>130</y><width>100</width><height>30</height></rect></property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry"><rect><x>100</x><y>170</y><width>200</width><height>20</height></rect></property>
    <property name="alignment">AlignCenter</property>
    <property name="text"><string/></property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>